src/npcmove.cpp

```diff
@@ -975,7 +975,7 @@ void npc::execute_action( npc_action action )
             // Find a nice spot to sleep
             int best_sleepy = sleep_spot( pos() );
             tripoint best_spot = pos();
-            for( const tripoint &p : closest_tripoints_first( pos(), 6 ) ) {
+            for( const tripoint &p : closest_points_first( pos(), 6 ) ) {
```

Comment:
This looks weird.  If I'm getting closest_points_first(), why aren't I iterating with a point?

Reply:
To allow `closest_points_first` to be used with the new generic point types, it's much easier if the same function name is used for both `point`s and `tripoint`s.  It's just choosing between them via function overloading.  In this case `pos()` is a `tripoint`, so you get `tripoint`s.